components/NavLayout/SideBar.js

```diff
@@ -1,12 +1,183 @@
 import styles from '../../utils/styles/NavLayoutStyles/SideBar.module.css';
```

Comment:
For imports, lets implement a styling of first importing in framework and library modules, newline, then modules we made. 
EX)
`import React from 'react
import MuiDrawer from '@mui/material/Drawer'

import CoolComponent from './nav/CoolComponent'
import styles from '../../utils/styles/someStyle,module.css'`

Reply:
Noted! Ill get right on that